thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,46 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetImageDiff(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetImageDiff")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+
+	// Verify that getImageDiff correctly returns Toolchain changes
+	opts := ImageDiffOptions{
+		BeforeAMI: "ami-029ab576546a58916",
+		AfterAMI:  "ami-02b25f680ad574d33",
+	}
+	result, err := c.getImageDiff(ctx, opts)
+	require.NoError(t, err)
+	assert.NotEmpty(result)
+	for _, change := range result {
+		assert.Equal(change.Type, "Toolchains")
```

Comment:
You can probably just have one test that has both Packages & Toolchains changes and do something like
```
	assert.True(change.Type == "Toolchains" || change.Type == "Packages")
```